dev-docs/conventions.md

```diff
@@ -1,39 +1,11 @@
-# Process conventions
+# Writing to customers: style policy
 
-## Pull request process
+Whenever we write text facing the customer (e.g docs, warnings, errors), we follow the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/welcome/).
+Some significant rules are these:
 
-Submissions should remain focused in scope and avoid containing unrelated commits.
-For pull requests, we employ the following workflow:
-
-1. Fork the repository to your own GitHub account
-2. Create a branch locally with a descriptive name
-3. Commit changes to the branch
-4. Write your code according to our development guidelines
-5. Push changes to your fork
-6. Clean up your commit history
-7. Open a PR in our repository and summarize the changes in the description
-
-## Reporting issues and bugs, asking questions
-
-This project uses the GitHub issue tracker. Please check the existing issues before submitting to avoid duplicates.
-
-To report a security issue, contact security@edgeless.systems.
-
-Your bug report should cover the following points:
-
-* A quick summary and/or background of the issue
-* Steps to reproduce (be specific, e.g., provide sample code)
-* What you expected would happen
-* What actually happens
-* Further notes:
-  * Thoughts on possible causes
-  * Tested workarounds or fixes
-
-## Major changes and feature requests
-
-You should discuss larger changes and feature requests with the maintainers. Please open an issue describing your plans.
-
-[Run CI e2e tests](/.github/docs/README.md)
+* [Avoid please](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please)
+* [Be brief](https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice#be-brief)
+* [Avoid we](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/we)
```

Comment:
Sorry, I didn't mean to indicate that these are especially significant rules in the other PR, but just the rules that applied to your change.
You may link to https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice for the most important rules.